plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryTasksProvider.kt

```diff
@@ -81,8 +85,39 @@ internal object SentryTasksProvider {
      * @return the provider if found or null otherwise
      */
     @JvmStatic
-    fun getMappingFileProvider(variant: ApplicationVariant): Provider<FileCollection> =
+    fun getMappingFileProvider(
+        project: Project,
+        variant: ApplicationVariant
+    ): Provider<FileCollection> = if (project.plugins.hasPlugin("com.guardsquare.proguard") ||
+        isDexguardAvailable(project)
+    ) {
```

Comment:
```suggestion
    ): Provider<FileCollection> = 
        if (project.plugins.hasPlugin("com.guardsquare.proguard") || isDexguardAvailable(project)) {
```
nit: I think this is a bit more readable